src/Okta.Idx.Sdk.E2ETests/Helpers/A18NClient/A18nClient.cs

```diff
@@ -305,17 +305,17 @@ private string GetRelativeUri(string resource = default)
             {
                 return $"{RelativePart}/{resource}";
             }
-        }
-
-        private void CleanUpOldUsedProfiles(string newProfileUniqueTag)
-        {
-            var activeProfiles = GetActiveProfilesAsync().Result;
-
-            foreach (var profile in activeProfiles.Profiles.Where(p => p.DisplayName == newProfileUniqueTag))
-            {
-                DeleteProfileAsync(profile.ProfileId).Wait();
-            }
-        }
+        }
+
+        private void CleanUpOldUsedProfiles(string newProfileUniqueTag)
+        {
+            var activeProfiles = GetActiveProfilesAsync().Result;
+
+            foreach (var profile in activeProfiles.Profiles.Where(p => p.DisplayName == newProfileUniqueTag))
```

Comment:
do we need a null check for profiles (`activeProfiles?.Profiles?`)?

Reply:
I don't see how they can be null, I expect this to be returned when there are no profiles available:
```json
{
    "profiles": [
    ]
}
```